runtime/test/strategies/strategy-test-helper.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2018 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+'use strict';
+
+import Arc from '../../arc.js';
+
+/** @class StrategyTestHelper
+ * Helper class holding utilities for testing strategies.
```

Comment:
This comment is redundant with the name of the class, seems unnecessary.

Reply:
Fixing.